### PR TITLE
5.2 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+## Release v5.2
+
+### Highlights
+
+#### Embed Mattermost in Other Apps (Beta)
+ - Added support for extensions, which allow you to embed Mattermost in other apps and websites via OAuth 2.0.
+ - A sample extension for Chrome [is here](https://github.com/mattermost/mattermost-chrome-extension). For documentation on how to write an extension, see here. // XXX Add a link to doc when ready
+
+#### Plugins
+ - Breaking changes to the plugins framework introduced. To migrate your existing plugins to be compatible with Mattermost 5.2 and later, see our [migration guide](https://developers.mattermost.com/extend/plugins/migration/).
```
Comment (esethna) [inline source/administration/changelog.md]: probably shouldn't lead with "breaking changes". Propose lead with the advantages and then mention breaking changes

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added support to add/delete and enable/disable plugins via the CLI.
+ - See our [demo plugin](https://github.com/mattermost/mattermost-plugin-demo) that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our [sample plugin](https://github.com/mattermost/mattermost-plugin-sample).
+ 
+#### Searching Archived Channels
+ - Added ability to search for archived channels.
```
Comment (esethna) [inline source/administration/changelog.md]: Might be worth mentioning "on desktop and mobile clients"

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ 
+#### Romanian Language
+ - Added support for Romanian language.
+
+### Improvements
+
+#### Web User Interface (UI)
+ - Added experimental custom default channels.
```
Comment (esethna) [inline source/administration/changelog.md]: Is this a config? If so it isn't under the config changes section. Do you have a link to more info on this as I'm not familiar with this change

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added link to profile pop-over from names in Join/Leave messages.
+ - Added autolinking to server markdown parser.
```
Comment (esethna) [inline source/administration/changelog.md]: Not sure what this means. Do you have a link to the PR or JIRA tix?

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Webapp Plugins: Beta
+ - Upcoming Mattermost UI redesign may cause breaking changes to webapp plugins. Hence, webapp plugins remain as beta in v5.2.
+ - Added support to override [...] post menu, and paperclip icon for file uploads.
+ - Added support for multiple plugins to add components at the same integration points instead of only allowing one plugin to do so.
+ - Removed ability to fully override profile popover. Instead, multiple plugins can now add to the profile popover via multiple integration points.
+ - For an up-to-date list of pluggable UI components, [see this list in our demo plugin](https://github.com/mattermost/mattermost-plugin-demo/tree/master/webapp#components).
+
+#### Notifications
+ - Added localized timestamp support for email notifications.
```
Comment (esethna) [inline source/administration/changelog.md]: I think this is only if you have the experimental timezone setting enabled.

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - In the compliance export status table, in System Console > Compliance > Compliance Export, added a number of exported records to Details column.
+ - Added support for cross-origin resource sharing.
+
+#### Command Line Interface (CLI)
+ - Enhanced log output from Permanent Delete CLI command to delete FileInfos for a user's posts.
+ - Addded channel renaming to CLI.
+ 
+#### Enterprise Edition
+ - Added the Global Relay Export CLI command.
+ - Added support to search message attachment/plugin contents.
```
Comment (esethna) [inline source/administration/changelog.md]: probably remove "message attachment" as that might get confused with file uploads

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### Bug Fixes
+ - Fixed an issue where the "Switch Channel" shortcut (⌘K) didn't work on dvorak layout on Mac.
```
Comment (esethna) [inline source/administration/changelog.md]: Are there more bug fixes coming?

File: source/administration/changelog.md (hunk 3 above)
Comment (amyblais) [reply source/administration/changelog.md]: @esethna Here the PR: https://github.com/mattermost/mattermost-server/pull/9068.

File: source/administration/changelog.md (hunk 4 above)
Comment (amyblais) [reply source/administration/changelog.md]: @esethna Ticket here: https://mattermost.atlassian.net/browse/MM-11451

**Comment by amyblais** [comment]
@cpanato to review / add API, Websocket and Database changes

File: source/administration/changelog.md (hunk 4 above)
Comment (esethna) [reply source/administration/changelog.md]: Not necessary for changelog I don't think

File: source/administration/changelog.md (hunk 3 above)
Comment (esethna) [reply source/administration/changelog.md]: Ah, this needs to be added to config changes compatibility section

Comment (esethna) [reply source/administration/changelog.md]: https://github.com/mattermost/mattermost-server/pull/9068/files#diff-2279f86634da3f787f279961af76713aR1108

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,6 +4,120 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added support for extensions, which allow you to embed Mattermost in other apps and websites via OAuth 2.0.
+ - A sample extension for Chrome [is here](https://github.com/mattermost/mattermost-chrome-extension). For documentation on how to write an extension, see here. // XXX Add a link to doc when ready
+
+#### Plugins
+ - Added support to add/delete and enable/disable plugins via the CLI.
+ - See our [demo plugin](https://github.com/mattermost/mattermost-plugin-demo) that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our [sample plugin](https://github.com/mattermost/mattermost-plugin-sample).
+ - Breaking changes to the plugins framework introduced. To migrate your existing plugins to be compatible with Mattermost 5.2 and later, see our [migration guide](https://developers.mattermost.com/extend/plugins/migration/).
+ 
+#### Searching Archived Channels
+ - Added ability to search for archived channels on desktop and mobile clients.
```
Comment (esethna) [inline source/administration/changelog.md]: Added ability to search for archived channel content on desktop and mobile clients.

File: source/administration/changelog.md (hunk 3 above)
Comment (amyblais) [reply source/administration/changelog.md]: @esethna It's not in the config file though https://github.com/mattermost/mattermost-server/blob/master/config/default.json.

**Comment by cpanato** [comment]
@amyblais for the database we added two new columns in the OutgoingWebhooks table https://github.com/mattermost/mattermost-server/pull/9216/files#diff-d7fac6dc5b7c8e3d295c8ab0c5cd5245R478

**Pull request merged**